docs/metastores.md

```diff
@@ -0,0 +1,98 @@
+```
+`conf-file` points to the EclipseLink configuration file and `persistence-unit` tells which unit from the configuration file to use for database connection.
+
+E.g., `polaris-dev` and `polaris` persistence units are configured in the same configuration file to connect to development database and production database respectively. Updating `persistence-unit` from `polaris-dev` to `polaris` switch from the development to the production.
+
+`conf-file` by default points to the embedded resource file `META-INF/persistence.xml` in `polaris-eclipselink` module.
+To specify an external configuration file,
+1) Place `persistence.xml` into a jar file: `jar cvf /tmp/conf.jar persistence.xml`.
+2) Use `conf-file: /tmp/conf.jar!/persistence.xml`.
+
+## EclipseLink Configuration - persistence.xml
+Configuration file `persistence.xml` is to configure the database connection properties, and the properties vary based on different databases and database setup.
+
+Check out [default persistence.xml](https://github.com/apache/polaris/blob/main/extension/persistence/eclipselink/src/main/resources/META-INF/persistence.xml) for the complete sample and the following shows the database connection properties against file-based H2 database. `Polaris` has a separate database for each realm. Internally `{realm}` in `jakarta.persistence.jdbc.url` is replaced by actual realm name so Polaris server connects to different database given the different realm.
+
+> Note: some database systems such as Postgres don't create databases automatically. Database admins need to create them manually before running Polaris server.
+```angular2html
+<persistence-unit name="polaris" transaction-type="RESOURCE_LOCAL">
+    <provider>org.eclipse.persistence.jpa.PersistenceProvider</provider>
+    <class>org.apache.polaris.core.persistence.models.ModelEntity</class>
+    <class>org.apache.polaris.core.persistence.models.ModelEntityActive</class>
+    <class>org.apache.polaris.core.persistence.models.ModelEntityChangeTracking</class>
+    <class>org.apache.polaris.core.persistence.models.ModelEntityDropped</class>
+    <class>org.apache.polaris.core.persistence.models.ModelGrantRecord</class>
+    <class>org.apache.polaris.core.persistence.models.ModelPrincipalSecrets</class>
+    <class>org.apache.polaris.core.persistence.models.ModelSequenceId</class>
+    <shared-cache-mode>NONE</shared-cache-mode>
+    <properties>
+      <property name="jakarta.persistence.jdbc.url"
+        value="jdbc:h2:file:tmp/polaris_test/filedb_{realm}"/>
+      <property name="jakarta.persistence.jdbc.user" value="sa"/>
+      <property name="jakarta.persistence.jdbc.password" value=""/>
+      <property name="jakarta.persistence.schema-generation.database.action" value="create"/>
+    </properties>
+  </persistence-unit>
+```
+
+Build with H2 dependency and run Polaris service:
+> polaris> ./gradlew --no-daemon --info -PeclipseLink=true -PeclipseLinkDeps=com.h2database:h2:2.3.232 clean shadowJar
```

Comment:
NIT: add one more block quotes (">") here so they will show in two lines instead of one line